Update meta info

do-you-dare opened the pull request:
This is part of https://github.com/mezuro/prezento/issues/350.

Commit:
Add CONTRIBUTING and HACKING meta files

Signed-off-by: Eduardo Silva Araújo <duduktamg@hotmail.com>

Commit:
Update README

Signed-off-by: Eduardo Silva Araújo <duduktamg@hotmail.com>

Commit:
Update changelog

Signed-off-by: Eduardo Silva Araújo <duduktamg@hotmail.com>

rafamanzo commented:
This build shall pass after merging https://github.com/mezuro/likeno/pull/24

rafamanzo commented on HACKING.md:
```diff
@@ -0,0 +1,26 @@
+  It is expected to work with:
+
+  - 2.1.5 Debian 8
+  - 2.0.0 CentOS 7
```
Those are no longer supported now. But we still can say 2.2.2 and further are.

Commit:
Remove older ruby references on HACKING file

Now, for the sake of Rails 5, support for rubies under 2.2.2 has been
dropped. This change reflects it.

rafamanzo commented on README.md:
```diff
@@ -72,4 +67,41 @@ It returns a `hash` with the response body.
+Copyright (c) 2013-2016 The Author developers.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
```
This uses the Lesser not Affero GPL :(

do-you-dare replied:
Sorry for that; I copied it from another repository from mezuro, tough I can't remember which one now.

Commit:
Fix license type

rafamanzo commented:
Nice job!
